api/v1alpha1/clustergroupupgrade_types.go

```diff
@@ -61,6 +61,9 @@ type ClusterGroupUpgradeSpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 	Clusters            []string                 `json:"clusters,omitempty"`
 	RemediationStrategy *RemediationStrategySpec `json:"remediationStrategy,omitempty"`
+	ManagedPolicy       []string                 `json:"managedPolicy,omitempty"`
+	//+kubebuilder:default=false
+	Enable bool `json:"enable,omitempty"`
```

Comment:
can you apply `$go fmt ` it will fix the style

Reply:
Heh, I did and that's what it did. I think it's like that because it's between //+kubebuilder markers.

Reply:
I believe if you re-order the struct fields having fields with annotation in top fmt will align them all.